src/main/java/hudson/plugins/jira/JiraReleaseVersionUpdaterBuilder.java

```diff
@@ -21,14 +21,16 @@
 
     private String jiraProjectKey;
     private String jiraRelease;
+    private String jiraDescription;
 
     @Extension
     public static final DescriptorImpl DESCRIPTOR = new DescriptorImpl();
 
     @DataBoundConstructor
-    public JiraReleaseVersionUpdaterBuilder(String jiraProjectKey, String jiraRelease) {
+    public JiraReleaseVersionUpdaterBuilder(String jiraProjectKey, String jiraRelease, String jiraDescription) {
```

Comment:
please always leave the previous constructor and marl it as deprecated (always backward compat)